### ISSUE TITLE
Student Council Page

### ISSUE THREAD

**Issue by chinathaip** [stamford-syntax-club/stamfordcenter-frontend#34]

**Comment by Ayham1590** [comment]
Student council page: 
- The page will be an anime poster of the student council team and can hover over each person to show the position.
- At the bottom, there will be for more info, and will have the past events and some pictures. 
- Then there will be a section for new upcoming events with posters which will be updated manually
- other ideas might come to mind but will think of it as designing.

**Comment by Ayham1590** [comment]
Lemme know what you think or if you wanna add more stuff, i will start designing before i code.

**Comment by chinathaip** [comment]
> * The page will be an anime poster of the student council team and can hover over each person to show the position.

Great idea, I'm not sure how the hovering thing will work out in mobile mode though. @Lxkas @NayHtetKyaw any input for this?

> * At the bottom, there will be for more info, and will have the past events and some pictures.

Cool, this section will be static right - simply hardcoded into the project


> * Then there will be a section for new upcoming events with posters which will be updated manually

How will the content be updated?

**Comment by Ayham1590** [comment]
The hover effect will not work on the mobile mode i guess.

We can update it everytime there's an upcoming event or something new from the student council. (I can do it its basically just replacing pictures)

**Comment by chinathaip** [comment]
I meant like, by changing the code content and submit pull request and all that?

**Comment by Ayham1590** [comment]
yes exactly like that

**Comment by chinathaip** [comment]
One solution I can offer is to have your page fetches data from our database and object storage server - this way it allows data to be dynamic and we don't have to rewrite the code every time we want to change something

For example: https://center-be-beta.stamford.dev/api/resources/resources

For your page it will be something like `https://center-be-beta.stamford.dev/api/resources/student_council`

The contents can be managed with our admin panel like this one: What do you think?
![image](https://github.com/stamford-syntax-club/stamfordcenter-frontend/assets/92321280/c71e6f5c-1710-4932-9c6d-22a7521e927c)

**Comment by Ayham1590** [comment]
Yea def doable, we can get on discord once i get to that part and you can help me through it maybe.

I've already started designing on figma.

**Comment by chinathaip** [comment]
Perfect! Keep us updated too

**Comment by Lxkas** [comment]
> > * The page will be an anime poster of the student council team and can hover over each person to show the position.
> 
> Great idea, I'm not sure how the hovering thing will work out in mobile mode though. @Lxkas @NayHtetKyaw any input for this?
> 
> > * At the bottom, there will be for more info, and will have the past events and some pictures.
> 
> Cool, this section will be static right - simply hardcoded into the project
> 
> > * Then there will be a section for new upcoming events with posters which will be updated manually
> 
> How will the content be updated?

Yeah, will not work on mobile. Do not rely on hover effects for functionality
